pages/mesosphere/dcos/1.13/release-notes/1.13.6/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.6
+title: Release notes for 1.13.6
+menuWeight: 2
+excerpt: Release notes for DC/OS 1.13.6, including Open Source attribution, and version policy.
+---
+DC/OS 1.13.6 was released on 7 November 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.13.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/1.13.6/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# Fixed and Improved Issues in DC/OS 1.13.6
+<!-- The issues that have been fixed and improved in DC/OS 1.13.6 are grouped by feature, functional area, or component.  -->
+- Fixed an issue where after upgrading to the latest version of MacOS Catalina, DC/OS certificates were identified as invalid. Fixed by updating the certificate dates. (DCOS-60264, DCOS-60205, COPS-5417)
```

Comment:
Let's remove the "Fixed by updating the certificate dates."  I am not sure it adds much info and it may cause more questions.

Reply:
Done.